joint_state_broadcaster/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>joint_state_broadcaster</name>
+  <version>0.2.0</version>
+  <description>Broadcaster to publish joint state</description>
+  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
```

Comment:
```suggestion
  <maintainer email="bence.magyar.robotics@gmail.com">Bence Magyar</maintainer>
  <maintainer email="denis@stogl.de">Denis Stogl</maintainer>
```

I would add also myself. @bmagyar is this OK? If you think that the list is overblown, you can leave me out :)

Reply:
if you want to get emails, so be it! :D